commons/src/main/java/fr/free/nrw/commons/category/CategorizationFragment.java

```diff
@@ -87,113 +95,72 @@ public void writeToParcel(Parcel parcel, int flags) {
         }
     }
 
-    private class CategoriesUpdater extends AsyncTask<Void, Void, ArrayList<String>> {
+    protected ArrayList<String> recentCatQuery() {
```

Comment:
This is fine but is this used in the class it's declared in? Should this be in its own class?